.gradlew/build is failing

**Actual Behaviour**

.gradlew/build command is failing on server setup

**Expected Behaviour**

It should not


**LogCat for the issue**

```
org.json.JSONObjectTest > test FAILED
    junit.framework.ComparisonFailure at JSONObjectTest.java:46
```

**Would you like to work on the issue?**
@akshatnitd would like to take up!

```
org.json.JSONObjectTest > test FAILED
    junit.framework.ComparisonFailure at JSONObjectTest.java:46
```
I am sending a fix for the above failure, which you were facing earlier. The failure mentioned in the issue description is because you had updated the dependencies to higher version, which the server doesn't support. Please move to the same version as before. Editing the issue, description with this error log.

Log sent by you - 
```
./gradlew build
Starting a Gradle Daemon, 1 incompatible Daemon could not be reused, use --status for details

> Task :compileJava
/Users/suneetsrivastava/Documents/Intern-Front/Susi/susi_server/src/api/external/transit/BahnService.java:10: error: package de.schildbach.pte does not exist
import de.schildbach.pte.*;
^
/Users/suneetsrivastava/Documents/Intern-Front/Susi/susi_server/src/api/external/transit/BahnService.java:11: error: package de.schildbach.pte.dto does not exist
import de.schildbach.pte.dto.Location;
                            ^
/Users/suneetsrivastava/Documents/Intern-Front/Susi/susi_server/src/api/external/transit/BahnService.java:12: error: package de.schildbach.pte.dto does not exist
import de.schildbach.pte.dto.QueryTripsResult;
                            ^
/Users/suneetsrivastava/Documents/Intern-Front/Susi/susi_server/src/api/external/transit/BahnService.java:13: error: package de.schildbach.pte.dto does not exist
import de.schildbach.pte.dto.Trip;
                            ^
/Users/suneetsrivastava/Documents/Intern-Front/Susi/susi_server/src/api/external/transit/BahnService.java:22: error: cannot find symbol
        private BahnProvider provider = new BahnProvider();
                ^
  symbol:   class BahnProvider
  location: class BahnService
/Users/suneetsrivastava/Documents/Intern-Front/Susi/susi_server/src/api/external/transit/JSONConverter.java:8: error: package de.schildbach.pte.dto does not exist
import de.schildbach.pte.dto.Fare;
                            ^
/Users/suneetsrivastava/Documents/Intern-Front/Susi/susi_server/src/api/external/transit/JSONConverter.java:9: error: package de.schildbach.pte.dto does not exist
import de.schildbach.pte.dto.Location;
                            ^
/Users/suneetsrivastava/Documents/Intern-Front/Susi/susi_server/src/api/external/transit/JSONConverter.java:10: error: package de.schildbach.pte.dto does not exist
import de.schildbach.pte.dto.Trip;
                            ^
/Users/suneetsrivastava/Documents/Intern-Front/Susi/susi_server/src/api/external/transit/JSONConverter.java:11: error: package de.schildbach.pte.dto.Trip does not exist
import de.schildbach.pte.dto.Trip.Leg;
                                 ^
/Users/suneetsrivastava/Documents/Intern-Front/Susi/susi_server/src/api/external/transit/JSONConverter.java:12: error: package de.schildbach.pte.dto does not exist
import de.schildbach.pte.dto.Line;
                            ^
/Users/suneetsrivastava/Documents/Intern-Front/Susi/susi_server/src/api/external/transit/JSONConverter.java:15: error: cannot find symbol
        public static JSONObject tripToJSON(Trip trip) {
                                            ^
  symbol:   class Trip
  location: class JSONConverter
/Users/suneetsrivastava/Documents/Intern-Front/Susi/susi_server/src/api/external/transit/JSONConverter.java:36: error: cannot find symbol
        public static JSONObject fareToJson(Fare fare) {
                                            ^
  symbol:   class Fare
  location: class JSONConverter
/Users/suneetsrivastava/Documents/Intern-Front/Susi/susi_server/src/api/external/transit/JSONConverter.java:48: error: cannot find symbol
        public static JSONObject legToJSON(Leg leg) {
                                           ^
  symbol:   class Leg
  location: class JSONConverter
/Users/suneetsrivastava/Documents/Intern-Front/Susi/susi_server/src/api/external/transit/JSONConverter.java:68: error: cannot find symbol
        public static JSONObject lineToJSON(Line line) {
                                            ^
  symbol:   class Line
  location: class JSONConverter
/Users/suneetsrivastava/Documents/Intern-Front/Susi/susi_server/src/api/external/transit/JSONConverter.java:80: error: cannot find symbol
        public static JSONObject locationToJSON(Location location) {
                                                ^
  symbol:   class Location
  location: class JSONConverter
/Users/suneetsrivastava/Documents/Intern-Front/Susi/susi_server/src/api/external/transit/JSONConverter.java:94: error: cannot find symbol
        public static String getDescription(List<Leg> legs) {
                                                 ^
  symbol:   class Leg
  location: class JSONConverter
16 errors


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileJava'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 7s
1 actionable task: 1 executed
static-50-50-50-179:susi_server suneetsrivastava$ 
```

--- a/test/org/json/JSONObjectTest.java
+++ b/test/org/json/JSONObjectTest.java
@@ -10,9 +10,9 @@ import org.junit.After;
 import org.junit.Before;
 
 public class JSONObjectTest extends TestCase {
-    
-    public static JSONObject testJson(boolean ordered) {
-        JSONObject json = new JSONObject(ordered);
+
+    public static JSONObject testJson() {
+        JSONObject json = new JSONObject();
         Map<String,  Object> map = new HashMap<>();
         map.put("abc", 1);
         map.put("def", "Hello World");
@@ -25,12 +25,12 @@ public class JSONObjectTest extends TestCase {
         json.put("fuenf", 5);
         return json;
     }
-    
+
     JSONObject testObject;
-    
+
     @Before
     public void setUp() throws Exception {
-        this.testObject = testJson(true);
+        this.testObject = testJson();
     }
 
     @After
@@ -45,5 +45,4 @@ public class JSONObjectTest extends TestCase {
         String t1 = j0.toString();
         assertEquals(t0, t1);
     }
-    
 }

